src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -100,6 +101,11 @@ def create_slack_client(config: SlackConversationConfiguration, run_async: bool
     return AsyncWebClient(token=config.api_bot_token.get_secret_value())
 
 
+def create_bolt_slack_app(config: SlackConversationConfiguration):
```

Comment:
The bolt app uses the `AsyncWebClient` I don't think we should create a whole bolt app just to use it's client. We have a function that creates a client for us to use in non "slack app" contexts (e.g. we just need to send a message based on the app itself, not based on some slack interaction). 

https://github.com/Netflix/dispatch/blob/e279cf6a49714d2d9b38eec9e21cb3d81dfbd815/src/dispatch/plugins/dispatch_slack/service.py#L17